lib/opFns.js

```diff
@@ -102,23 +102,21 @@ module.exports = {
     }
   },
   SIGNEXTEND: function (k, val, runState) {
-    val = val.toArrayLike(Buffer, 'be', 32)
-    var extendOnes = false
-
     if (k.lten(31)) {
```

Comment:
This should be ``ltn(31)` or `lten(30)`.

Reply:
This part needs to be verified, not sure where this comment came from, but if state tests are passed with the current one that can only mean two things: a) it is correct or b) state tests are lacking.